analyser/src/main/java/io/polyrepo/analyser/service/IssueService.java

```diff
@@ -64,5 +64,53 @@
             return new ResponseEntity<>(Collections.singletonMap("message","Bad Request"),HttpStatus.BAD_REQUEST);
         }
     }
+
+    /**This method returns the list of priority-1 issue creation and closing time through
+     * which average time taken to resolve priority-1 issues can be obtained
+     * @param orgUserName GitHub Organization login name
+     * @param token       GitHub personal access token
+     * @return            List of priority-1 issues' creation and closing time
+     */
+    public ResponseEntity<?> getClosedP1IssuesTime(String orgUserName, String token) {
```

Comment:
why is service class returning `ResponseEntity`? Service classes are not supposed to deal with Request and Responses. They should deal with Entities.